### DISCUSSION TITLE
What is the minimum nuber of CPUs?

### DISCUSSION THREAD

**Discussion by maflcko** [cirruslabs/cirrus-ci-docs#872] [Q&A]
I created a test task with `0.0625 CPUs / 0.01953125 GBs` and it passed: https://cirrus-ci.com/task/6058979882172416

**Comment by fkorotkov** [comment]
For the non-privileged `container`s min CPU value can be any. The only rule there is 1:4 ration on cpu/memory.

For VMs and privileged containers the CPUs should be integers.
